reyna/src/com/b2msolutions/reyna/blackout/Time.java

```diff
@@ -39,4 +39,8 @@ public boolean isBeforeOrEqualTo(Time other) {
     public boolean isAfterOrEqualTo(Time other) {
         return this.minuteOfDay >= other.minuteOfDay;
     }
+
+    public long getCurrentTimeMillis() {
+        return System.currentTimeMillis();
```

Comment:
Why are we using blackout/time for this? It has nothing to do with it, has it? Can we not just call System.currentTimeMillis() directly in the dispatcher? If we really want to use blackout class for this (which I don't think we should), should this function not be static (or however that is called in Java, i.e. not require an instance of the class, but called from class itself)?